inspector/hardhat.config.ts

```diff
@@ -73,4 +82,105 @@ const config: HardhatUserConfig = {
   },
 };
 
+task("inspect", "Main task")
```

Comment:
Tasks do not have to be defined within the `hardhat.config.ts`. How about defining it in separate file to keep it less packed here?

Check how LayerZero is doing it :)

https://github.com/LayerZero-Labs/LayerZero/tree/main/tasks